playground/CMakeLists.txt

```diff
@@ -1,22 +1,8 @@
-PROJECT(playground)
+add_executable( stlab.playground main.cpp )
 
-set(SOURCE
-  main.cpp
-)
+target_link_libraries( stlab.playground
+                       PUBLIC ${Boost_LIBRARIES} 
+                       PUBLIC stlab )
 
-set(HEADERS
-)
-
-add_executable(playground ${SOURCE} ${HEADERS})
-
-include_directories(${Boost_INCLUDE_DIRS})
-
-target_link_libraries(
-    playground
-    ${Boost_LIBRARIES} 
-    ${CMAKE_THREAD_LIBS_INIT} 
-    ${CONAN_LIBS})
-
-
-set_property(TARGET playground PROPERTY CXX_STANDARD 14)
-set_property(TARGET playground PROPERTY CXX_STANDARD_REQUIRED ON)
+set_property( TARGET stlab.playground PROPERTY CXX_STANDARD 14 )
```

Comment:
What's the difference between setting the property for the target as you are doing here, versus using the compiler-targeted `include` cmake/`set` statements that you did at the top level? Would these sub-project(s) benefit from what you did up top?